<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAddPlatformDialog</class>
 <widget class="QDialog" name="CAddPlatformDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_ptxtAPIURL"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>API URL</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="m_ptxtSecKey"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sec Key</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>API Key</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_ptxtAPIKey"/>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_pbtnOK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pbtnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Platform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_ptxtPlatformName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Class Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_ptxtClassName"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_ptxtPlatformName</tabstop>
  <tabstop>m_ptxtClassName</tabstop>
  <tabstop>m_ptxtAPIURL</tabstop>
  <tabstop>m_ptxtAPIKey</tabstop>
  <tabstop>m_ptxtSecKey</tabstop>
  <tabstop>m_pbtnOK</tabstop>
  <tabstop>m_pbtnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
